modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -668,7 +668,7 @@ AUI.add(
 
 				var secondOperandTypeValue = instance._getSecondOperandTypeValue(index);
 
-				var visible = instance._isConstant(secondOperandTypeValue) && !instance._isFieldList(firstOperand);
+				var visible = instance._isText(secondOperandTypeValue) && !instance._isFieldList(firstOperand);
```

Comment:
group these expressions with parentheses and if you don't use in anywhere else without being in that `if` expressions bellow, you can remove this line and parse this expression inside the `if`

Reply:
It's ok. But the "visible" variable is being used outside the "if" condition, so I'll keep it.